.storybook/test-runner.ts

```diff
@@ -0,0 +1,31 @@
+import { injectAxe, checkA11y } from 'axe-playwright';
+import { Page } from 'playwright-core';
+
+/*
+* See https://storybook.js.org/docs/react/writing-tests/test-runner#test-hook-api-experimental
+* to learn more about the test-runner hooks API.
+*/
+const renderFunctions = {
+  async preRender(page: Page) {
+    await injectAxe(page);
+  },
+  async postRender(page: Page, context) {
+    await checkA11y(page, '#root', {
+      axeOptions: {
+        runOnly: {
+          type: 'tag',
+          values: ['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa']
+        },
+        rules: {
+          'color-contrast': { enabled: context.name !== 'Loading' }
+        },
```

Comment:
we also defined this in preview.js (excluding rules here, which we should probably add to preview.js), can we have a file containing this configuration that is use in both preview and test-runner?

Reply:
I moved the common part to another file, but as you mentioned, the rules can't go in preview.js because we don't have context there for the story's name. we would have to configure that individually for each Loading story